src/shared/platforms/linux/sources.cmake

```diff
@@ -9,3 +9,5 @@ target_include_directories(shared-sources INTERFACE ${libsecret_INCLUDE_DIRS})
 target_sources(shared-sources INTERFACE
      ${CMAKE_CURRENT_SOURCE_DIR}/shared/platforms/linux/linuxcryptosettings.cpp
 )
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/shared/signature.cmake)
```

Comment:
`signature.cmake` was replaced by a generalized Rust helper, and you should now do something like the following:

```
include(${CMAKE_SOURCE_DIR}/scripts/cmake/rustlang.cmake)
add_rust_library(signature
    PACKAGE_DIR ${CMAKE_SOURCE_DIR}/signature
    BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}
    CRATE_NAME signature
)
target_compile_definitions(mozillavpn PRIVATE MZ_SIGNATURE)
target_link_libraries(mozillavpn PRIVATE signature)
```